clash-cores/test/Test/Cores/I2C/Config.hs

```diff
@@ -22,8 +22,18 @@ data ConfS = ConfS { i2cConfStateM   :: ConfStateMachine
                    , i2cConfFault    :: Bool
                    }
 
-type ConfI = (Bool,Bool,Bool,Bool,Bool)
-type ConfO = (Bool,Maybe I2COperation,Bool,Bool)
+data ConfI = ConfI { i2cRst    :: Bool
```

Comment:
The commit message just says "refactor", perhaps explain what the refactor does.